packages/contracts/src/typechain-types/IAirdrop.ts

```diff
@@ -37,15 +36,15 @@ export interface IAirdropInterface extends Interface {
 
   encodeFunctionData(
     functionFragment: "claim",
-    values: [BigNumberish, BigNumberish, BigNumberish, BytesLike[]]
+    values: [bigint, bigint, bigint, BytesLike[]]
```

Comment:
So I assume we want all these `typechain-types` to use `bigint`

Reply:
No, `typechain` is required only for `ethers` because `ethers` does not have built-in support for Typescript. Once we make the transition, we will be getting rid of `typechain` entirely. `viem` has built-in TS support: https://viem.sh/docs/contract/getContract

These files are autogenerated so should not be updated. We should only remove any `BigNumberish` imports that we reference within our own code